components/ota.rst

```diff
@@ -1,59 +1,23 @@
-ESPHome OTA Updates
-===================
+Over-the-Air Updates
+====================
 
 .. seo::
-    :description: Instructions for setting up ESPHome's Over-The-Air (OTA) platform to allow remote updating of devices.
+    :description: Instructions for setting up the Over-The-Air (OTA) component to allow remote updating of devices.
     :image: system-update.svg
 
 .. _config-ota:
 
-ESPHome's Over-The-Air (OTA) platform allows you to remotely install modified/updated firmware binaries onto your
-ESPHome devices over their network (Wi-Fi or Ethernet) interface.
+ESPHome supports remotely updating a device "over-the-air" (OTA). Each update mechanism is a *platform* of the base
+``ota`` component and will have its own configuration variables.
 
-This platform is used by both the ESPHome dashboard as well as the command line interface (CLI) (via
-``esphome run ...``) to install firmware onto supported devices.
-
-In addition to OTA updates, ESPHome also supports a "safe mode" to help with recovery if/when updates don't work as
-expected. This is automatically enabled by this component, but it may be disabled if desired. See
-:doc:`/components/safe_mode` for details.
+In release 2024.6.0, the ``ota`` component transistioned from a standalone component to a *platform* component. This
+change was made to facilitate the use of multiple update mechanisms, enabling greater flexibility.
 
 .. code-block:: yaml
 
     # Example configuration entry
     ota:
-      - platform: esphome
-        safe_mode: true
-        password: !secret ota_password
-
-Configuration variables:
-------------------------
-
--  **password** (*Optional*, string): The password to use for updates.
--  **port** (*Optional*, int): The port to use for OTA updates. Defaults:
-
-   - ``3232`` for the ESP32
-   - ``8266`` for the ESP8266
-   - ``2040`` for the RP2040
-   - ``8892`` for Beken chips
--  **id** (*Optional*, :ref:`config-id`): Manually specify the ID used for code generation.
--  **on_begin** (*Optional*, :ref:`Automation<automation>`): An action to be performed when an OTA update is started.
-   See :ref:`ota-on_begin`.
--  **on_progress** (*Optional*, :ref:`Automation<automation>`): An action to be performed (approximately each second)
-   while an OTA update is in progress. See :ref:`ota-on_progress`.
--  **on_end** (*Optional*, :ref:`Automation<automation>`): An action to be performed after a successful OTA update.
-   See :ref:`ota-on_end`.
--  **on_error** (*Optional*, :ref:`Automation<automation>`): An action to be performed after a failed OTA update.
-   See :ref:`ota-on_error`.
--  **on_state_change** (*Optional*, :ref:`Automation<automation>`): An action to be performed when an OTA update state
-   change happens. See :ref:`ota-on_state_change`.
```

Comment:
These should stay here as they are common to all platforms...

Reply:
Ugh, totally forgot to paste this section back in 🤦🏻